src/data/navigation.json

```diff
@@ -6,5 +6,9 @@
   {
     "href": "/team",
     "title": "team"
+  },
+  {
+    "href": "/projects",
```

Comment:
Why is this back in the navigation?

Reply:
It should't be in it in the final merge. And in this commit [
13d75ef](https://github.com/rightinyourwheelhouse/wheelhouse-website/pull/137/commits/13d75ef43a8c369412cebc9028892607ef5a71ee) it is deleted.